src/pages/AppDetail.svelte

```diff
@@ -11,9 +12,33 @@
   });
 </script>
 
+<style>
+  .table-header {
+    @apply table-auto;
+    @apply my-4;
+  }
+
+  .table-header td {
+    @apply border;
+    @apply p-2;
+  }
+</style>
+
 <h1>{params.app}</h1>
 {#if app}
-  <p>{app.description}</p>
+  <p class="mb-2">{app.description}</p>
+  <MetaData data={app} />
```

Comment:
Two things: 

1. This should only display if the deprecated property is true. Also we should just show "Deprecated".
2. It should be above the description